packages/frontend/component/src/lit-react/lit-portal/lite-portal.tsx

```diff
@@ -98,21 +98,29 @@ export const useLitPortalFactory = () => {
 
   return [
     useCallback(
-      (elementOrFactory: React.ReactElement | (() => React.ReactElement)) => {
+      (
+        elementOrFactory: React.ReactElement | (() => React.ReactElement),
+        nonRerendering?: boolean
```

Comment:
Where do you use this parameter

Reply:
<img width="938" alt="Screenshot 2024-12-25 at 11 33 54" src="https://github.com/user-attachments/assets/46265505-4107-4511-babe-40724cb86ce7" />